.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,39 @@
+---
+name: 🙋 Feature Request
+about: Want us to add something to FAST?
+title: "feat: add [what] to/in [where]"
+labels: 'status:triage'
+---
+
+<!---
+Thanks for filing an issue 😄 ! Before you submit, please read the following:
```

Comment:
Is there a way to templatize the duplicative content that used across templates?

Reply:
Not that I'm aware of. I read all the docs on issue and PR templates and nothing comes up for that.